cbmgmtx/mgmt.go

```diff
@@ -98,6 +108,33 @@ func (h Management) DecodeCommonError(resp *http.Response) error {
 	}
 }
 
+type InvalidArg struct {
```

Comment:
I wonder if this should just be ServerInvalidArgError, which unwraps to ErrServerInvalidArg. Then errors.As can be used to access field and reason if we make them public instead.

Reply:
+1

Reply:
Done!